content/integrations/sso/sso-jumpcloud.md

```diff
@@ -31,9 +31,9 @@ Keep this tab open and open a new tab; you must now login to JumpCloud and uploa
 3. Search for `SAML` and select **Custom SAML App** > **Configure**.
 4. Input the following values:
   - **DISPLAY LABEL**: `Your SSO Label` ("A name of your choosing to identify this configuration.)
-  - **IDP ENTITY ID**: `jumpcloud-metricly`
+  - **IDP ENTITY ID**: `jumpcloud-cloudwisdom`
```

Comment:
@rclagett please confirm this would work.

Reply:
I just re-watched our SSO vid for the IDP entity ID and it's an arbitrary field. I could just remove it?

Reply:
No, I think it's good as is.